pelican/plugins/search/search.py

```diff
@@ -15,6 +15,7 @@
 import subprocess
 
 from jinja2.filters import do_striptags as striptags
+from json import dumps
```

Comment:
Did you [set up your plugin dev environment](https://docs.getpelican.com/en/latest/contribute.html#plugin-development) and then run the linters locally before committing and pushing? I only ask because CI is showing a linter failure due to disordered import statements.

Reply:
I have now (and thanks for the coaching!) but something in that recipe is seriously at odds with my system. Now every commit tries to reinstall poetry, invoke, etc., then encounters a compatbility error and aborts. I'll follow up with a seperate issue after I've had a chance to try and understand what's going on. For now I'll just edit the file manually via Github.